src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/SFA.DAS.EmployerIncentives.Functions.LegalEntities.csproj

```diff
@@ -11,7 +11,8 @@
     <PackageReference Include="NServiceBus" Version="7.1.7" />
     <PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="3.0.84" />
     <PackageReference Include="SFA.DAS.EmployerAccounts.Messages" Version="1.6.2978" />
-    <PackageReference Include="SFA.DAS.EmployerIncentives.Messages" Version="0.1.25" />
+    <PackageReference Include="SFA.DAS.EmployerIncentives.Messages" Version="0.1.61-EI-279-VRF-status-base.20-prerelease" />
```

Comment:
Does this version need updating?

Reply:
It will do but I can't generate a proper version until the das-employerincentives changes are in master.